### PR TITLE
Update limitation for js

### PR DESCRIPTION
**Pull request by kaibocai**
Update limitation for js

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md [hunk 1]
```diff
@@ -247,11 +268,29 @@ HttpResponse<String> response = HttpClient.newBuilder().build().send(request, Ht
 
 {{% /codetab %}}
 
+```javascript
```
Comment (hhunter-ms) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md:273]:
```suggestion
{{% codetab %}}

```javascript
```

Comment (hhunter-ms) [reply daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md:273]: this should fix the build error in the PR

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
Could you also change this PR to point to branch `v1.13`?

**Comment by msfussell** [comment]
@kaibocai - Ping on the above request.

**Review by hhunter-ms — APPROVED** [review]

**Pull request merged**